tor/onionfile/onion_file_test.go

```diff
@@ -0,0 +1,97 @@
+package onionfile
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+)
+
+var (
+	privateKey = []byte("RSA1024 hide_me_plz")
```

Comment:
Any reason this isn't just `hide_me_plz` like it was before?

Reply:
Ah yup so that's added as a way to tell if the key is encrypted or not, see this line https://github.com/lightningnetwork/lnd/pull/6526/files#diff-a219f7652057175a72ef2276ada3396923cfd0c3e82b4149bd1705cb2a70a9d5R111

Reply:
Gotcha, makes sense!